src/xdocs/checks/blocks/rightcurly.xml

```diff
@@ -172,33 +175,37 @@ public class Test {
         </source>
         <p id="Example2-code">Example:</p>
         <source>
-public class Test {
+public class Example2 {
 
   public void test() {
 
+    boolean foo = false;
     if (foo) {
       bar();
-    } else { bar(); }   // violation, right curly must be alone on line
+    } else { bar(); } // violation, ''}' at column 21 should be alone on a line.'
 
     if (foo) {
       bar();
     } else {
       bar();
-    }                   // OK
+    }
 
     try {
       bar();
-    } catch (Exception e) { // OK because config is set to token METHOD_DEF and LITERAL_ELSE
+    } catch (Exception e) { // OK because config is set*
```

Comment:
`*` does not matter  in "//ok" comments, pelase restore content.

Reply:
done @romani